samcli/commands/sync/command.py

```diff
@@ -158,6 +158,14 @@
     help="This option will skip the initial infrastructure deployment if it is not required"
     " by comparing the local template with the template deployed in cloud.",
 )
+@click.option(
+    "--container-env-var-file",
+    "-ef",
+    default=None,
+    type=click.Path(),  # Must be a json file
+    help="Environment variables json file (e.g., env_vars.json) to be passed to containers.",
+    cls=ContainerOptions,
+)
```

Comment:
We usually keep common options in same place to reduce duplication. One example can be found here; https://github.com/aws/aws-sam-cli/blob/232f88acc3a1ccc2056eab4d0490ff710f4a9674/samcli/commands/_utils/options.py#L798-L813

Would you mind moving this option to there as well and use it in both places?

Reply:
yes sure